src/main/src/main/java/com/shevtsov/Creator.java

```diff
@@ -0,0 +1,30 @@
+package com.shevtsov;
+
+import com.shevtsov.services.Authorisation;
+import com.shevtsov.services.ClientService;
+import com.shevtsov.services.OrderService;
+import com.shevtsov.services.ProductService;
+import com.shevtsov.services.impl.AuthorisationImpl;
+import com.shevtsov.services.impl.ClientServiceImpl;
+import com.shevtsov.services.impl.OrderServiceImpl;
+import com.shevtsov.services.impl.ProductServiceImpl;
+import com.shevtsov.validators.ValidationService;
+import com.shevtsov.validators.impl.ValidationServiceImpl;
+import com.shevtsov.view.*;
+
+public class Creator {
+    public static MainMenu create(){
+        ValidationService validationService = new ValidationServiceImpl();
+        ClientService clientService = new ClientServiceImpl(validationService);
+        ProductService productService = new ProductServiceImpl();
+        OrderService orderService = new OrderServiceImpl();
```

Comment:
why you mixed constructors and static methods?

Reply:
to not create an instance of the class Create. is this a bad idea? why?

Reply:
corrected